.github/actions/aws-ci/action.yml

```diff
@@ -0,0 +1,74 @@
+name: 'AWS CI'
+description: 'AWS integration tests'
+inputs:
+  elk-version:
+    description: 'ELK version'
+    required: true
+  aws-access-key-id:
+    description: 'AWS access key id'
+    required: true
+  aws-secret-access-key:
+    description: 'AWS secret access key'
+    required: true
+  aws-account-type:
+    description: 'AWS account type'
+    required: false
+    default: single-account
+runs:
+  using: composite
+  steps:
+    - name: Build cloudbeat binary
+      shell: bash
+      run: mage -v build
+
+    - name: Run elasticsearch
+      uses: elastic/elastic-github-actions/elasticsearch@master
+      with:
+        stack-version: ${{ inputs.elk-version }}
+        security-enabled: false
+
+    - name: Elasticsearch disable dynamic mapping for findings
+      shell: bash
+      run: ./.ci/scripts/disable_dynamic_mapping.sh
```

Comment:
@moukoublen These actions are repeated in all testing CIs. Perhaps we should consider creating a common action for this as well?

Reply:
Sure, I will create something like `run-elastisearch`